jep/0000/README.adoc

```diff
@@ -0,0 +1,145 @@
+* Experience - the candidate should point to experience with Twitter/social media or submit sample tweets
+
+Once approved, the candidate should share their twitter account with the Twitter account owner
+The Twitter owner will invite the candidate to join the account Team via Tweetdeck. 
+
+New admins will typically be experienced contributors who the board approve as administrators. 
+
+==== Responsibilites of Contributors
+
+Admin and Contributors will make regular tweets or retweets to meet the goals below.
+Admin and Contributors agree to abide by the Jenkins Twitter policy. 
+The Twitter owner may revoke access if contributors violate this policy. 
+If contributors are inactive they may request to leave the team or the owner may revoke access. 
+
+=== Jenkins Twitter Policy
+
+==== Goals
+The goals of the @jenkinsci twitter account are:
+
+* Create awareness of Jenkins projects, releases, plugins and features
+* Promote awareness of Jenkins working groups, events, JAMs or talks about Jenkins
+* Build awareness of Jenkins and encourage contributions
+* Celebrate achievements of the community
+* Grow the open source community around Jenkins
+* Always be a trustworthy, reputable source of information
+
+==== Best Practices
+These are the best practices for contributors to follow:
+
+* Conduct - always abide by Jenkins community code of conduct .
+* Content - write or curate tweets that meet one of the goals above. 
+* Cadence - contributors should work together to maintain a regular cadence e.g. 3-5 tweets a day
+* Lead with community - tweets should serve a purpose to the community first.
+* Credit - wherever possible give credit to author, project or originating organisation.
+* Hashtags - use these to link themes or help community find content. Ideally max 2 hashtags per tweet.
+* Voice - be community facing, share a common tone with other contributors avoid corporate tones.
+* Support - avoid being a support channel for the Jenkins project
+
+=== Regular Reporting 
+The Twitter account owner will download monthly analytics reports from Twitter.
+These will be shared with the contribution team. 
+One member of the contribution team will prepare a quarterly report to be shared with the community summarizing stats and insights. 
+
+== Motivation
+
+Jenkins twitter accounts have a massive following. 
+To date there is usually one owner tweeting, retweeting and all other twitter activity. 
+There are automated tweets but these have some limitations e.g no tags, no images included in tweets. 
+These policies aim to lighten the burden on Twitter owners and use twitter to meet Jenkins project goals
+
+== Reasoning
+Having a well defined policy enables the community to welcome contributors and share best practices. 
+
+
+== Backwards Compatibility
+
+There are no backwards compatibility concerns related to this proposal.
+
+
+== Security
+
+Contribution team members are recommended to use login verification on their twitter accounts.
```

Comment:
No experience with this, but shouldn't this just be a requirement, i.e. mandatory?

Reply:
There's no way to enforce this properly via TweetDeck

Reply:
Will make this a requirement, but yes no way to enforce.